doc/dev/cephfs-snapshots.rst

```diff
@@ -53,6 +53,17 @@ To create a CephFS snapshot, create a subdirectory under
 ``.snap`` with a name of your choice. For example, to create a snapshot on
 directory "/1/2/3/", invoke ``mkdir /1/2/3/.snap/my-snapshot-name`` .
 
+.. note::
+   Snapshot names can not start with an underscore ('_'), as these names are
+   reserved for internal usage.
+
+.. note::
+   Snapshot names can not exceed 240 characters.  This is because the MDS makes
+   use of long snapshot names internally, which follow the format:
+   `_<snapshot-name>_<node-id>`.  Since filenames in general can't have more
```

Comment:
We'd better use the same format example as `_<SNAPSHOT-NAME>_<INODE-NUMBER>`. 
The others look good to me.

Reply:
Yeah, that looks better.  I'll update my branch in a bit with this change.  Thanks @lxbsz